lib/routes/aves/index.ts

```diff
@@ -0,0 +1,93 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/:category?',
+    categories: ['other'],
+    example: '/aves',
+    parameters: { category: '分类，见下表' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '分类',
+    maintainers: ['dddaniel1'],
+    handler,
+    description: `| 诗歌 | 小说 | 专栏 | 档案 | 非虚构 | all | current
+  | -------- | -------- | -------- | -------- | -------- |
+  | 1     | 2     | 3     | 4     | 5     | all     | 最新文章     |`,
```

Comment:
Missing pipe in table header.
No. of row separator in does not match with the content.